### DISCUSSION TITLE
Follow the single coding style.

### DISCUSSION THREAD

**Discussion by lserveriiev** [9tigerio/db2rest#816] [General]
At this moment each contributor follows own code style. I do propose to follow the [Google Java Style](https://google.github.io/styleguide/javaguide.html) as a project's code style.

**Comment by kdhrubo** [comment]
Go for it. What's the setting ?

**Comment by lserveriiev** [comment]
It can be downloaded from [here](https://github.com/google/styleguide/blob/gh-pages/intellij-java-google-style.xml). I suspect we need to update the contribution info. Create an issue and assign it to me, please.
